codalab/model/bundle_model.py

```diff
@@ -821,39 +812,40 @@ def resume_bundle(self, bundle, bundle_update, row, user_id, worker_id, hostname
             worker_run_row = {'user_id': user_id, 'worker_id': worker_id, 'run_uuid': bundle.uuid}
             connection.execute(cl_worker_run.insert().values(worker_run_row))
 
-        metadata_update = {
-            'run_status': bundle_update['run_status'],
-            'last_updated': int(time.time()),
-            'time': time.time() - bundle_update['start_time'],
+        bundle_update = {
+            'state': worker_bundle_update['state'],
+            'metadata': {
+                'run_status': worker_bundle_update['run_status'],
+                'time': worker_bundle_update['time'],
+                'time_user': worker_bundle_update['time_user'],
+                'time_system': worker_bundle_update['time_system'],
+            }
         }
 
-        if bundle_update['docker_image'] is not None:
-            metadata_update['docker_image'] = bundle_update['docker_image']
+        if worker_bundle_update['docker_image'] is not None:
+            bundle_update['metadata']['docker_image'] = worker_bundle_update['docker_image']
 
         self.update_bundle(
-            bundle, {'state': bundle_update['state'], 'metadata': metadata_update}, connection
+            bundle, bundle_update, connection
         )
 
         self.update_events_log(
             user_id=bundle.owner_id,
             user_name=None,  # Don't know
-            command='resume_bundle',
```

Comment:
I don't know how events_log works but let's make sure we don't break backwards compatibility here. @percyliang thoughts?

Reply:
This seems OK!